frontend/javascripts/dashboard/dataset/dataset_import_view.js

```diff
@@ -174,54 +288,125 @@ class DatasetImportView extends React.PureComponent<Props, State> {
       Modal.info({
         title,
         width: 800,
-        content: (
-          <Input.TextArea rows={20} style={jsonEditStyle}>
-            {jsonStringify(object)}
-          </Input.TextArea>
-        ),
+        content: <Input.TextArea rows={20} style={jsonEditStyle} value={jsonStringify(object)} />,
```

Comment:
I had to make this change after merging the newest master into this branch. Otherwise, the text area would be empty. 

I do not know why this change is needed. Maybe antd was updated? But at least it works :)

Reply:
I just noticed that antd indeed was upgraded